ghost/core/core/server/web/api/endpoints/admin/middleware.js

```
@@ -64,14 +64,16 @@ const notImplemented = function notImplemented(req, res, next) {
         files: ['POST'],
         media: ['POST'],
-        db: ['POST'],
+        db: ['GET', 'POST'],
         settings: ['GET'],
         oembed: ['GET'],
         'search-index': ['GET']
     };
 
     const match = req.url.match(/^\/([^/?]+)\/?/);
 
+    console.log('match', match, req.path, req.method);
```

ESLint

[error] Unexpected console statement.

(no-console)